Resources/Prototypes/Entities/Objects/Devices/pda.yml

```
@@ -622,6 +622,15 @@
     borderColor: "#6f6192"
   - type: Icon
     state: pda-lawyer
+  - type: CartridgeLoader
```

Move comment from above to here (assuming those are the only two added, compared to upstream)

```suggestion
  - type: CartridgeLoader # DeltaV - Add CrimeAssist and LogProbe cartridges
```